styles/components/shared/infowindow.scss

```diff
@@ -44,3 +44,54 @@
     }
   }
 }
+
+.c-profile-tooltip {
```

Comment:
There's a lot of repetition with above `.c-sankey-tooltip` component. Try to see if merging both into a generic infowindow would not be too complicated.